CI/install_vcpkg_dependencies.sh

```diff
@@ -1,7 +1,17 @@
 #!/usr/bin/env bash
 
-RELEASE_TAG="v1.8"
-FILENAME="dependencies-$1"
-DOWNLOAD_URL="https://github.com/vcmi/vcmi-deps-windows/releases/download/$RELEASE_TAG/$FILENAME.txz"
+REPO="vcmi"
 
-curl -L "$DOWNLOAD_URL" | tar -xf - --xz
+# Fetch latest release tag from GitHub API
+RELEASE_TAG=$(curl -s "https://api.github.com/repos/$REPO/vcmi-deps-windows/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
```

Comment:
This may break builds in future.

For example, let's assume that we removed library X, and created new release without that library.

When this happens, all CI in vcmi would fail (other than PR with removal of said library) - both daily builds and all PR's.

With current approach CI always points to *correct* release.